libc/nt/struct/cpinfoex.h

```diff
@@ -0,0 +1,17 @@
+#ifndef COSMOPOLITAN_LIBC_NT_STRUCT_CPINFOEX_H_
+#define COSMOPOLITAN_LIBC_NT_STRUCT_CPINFOEX_H_
+
+#define MAX_LEADBYTES    12 
```

Comment:
I recommend just inlining these values and not defining constants for them. Definitions are only useful if they can be changed. But these numbers are a permanent part of this ABI. Do you actually need them in your source code? If so, possibly put them in windowsesque.h? Please note `PATH_MAX` in cosmo is defined as 1024 and our `__mkntpath()` function will actually insert `\\?\` when absolutely necessary to overcome the 260 character limit.

Reply:
I use them in my source code but I finally choose to inline them because I noticed that `MAX_PATH` was inlined in `processentry32.h` so for the sake of consistency I did the same.